app/views/topics/new.html.erb

```diff
@@ -35,7 +35,8 @@
   <!--<label>Tags:</label><br/>-->
   <%#= text_field_tag :tags %>
 
-  <%= f.submit t(:submit_start_discussion, default: 'Start Discussion'), class: 'btn btn-warning disableable', disabled: 'disabled' %>
+  <%= f.submit t(:submit_start_discussion, default: t('submit_start_discussion')), class: 'btn btn-warning disableable', disabled: 'disabled' %>
   </div>
```

Comment:
The default provides backup text in case the i18n files don't have the translation.  Putting the translation key there again would probably cause an error if the string was not translated?  I think this needs to stay the same as it was.